include/nudb/basic_store.hpp

```diff
@@ -236,8 +236,17 @@ class basic_store
 
     /** Fetch a value.
 
-        where data and size represent the value. If the
-        key is not found, the handler is not called.
+        The function checks the database for the specified
+        key, and invokes the callback if it is found. If
+        the key is not found, `ec` is set to @ref error::key_not_found.
+        If any other errors occurs, `ec` is set to the
```

Comment:
Assuming that if an error occurs ec is set and returned immediately, this should probably read
error occurs